web-console/knikubevirt/Create-vm/2.networking/create-vm-networking.md

```diff
@@ -0,0 +1,20 @@
+# Create A Virtual Machine
+
+## Step 2: Networking
+
+![Default](img/3-00.png)
+The user enters the Networking step with an existing NIC pre-defined, set to Type (local). In this step, the 'Review and Create' button is still available to skip the rest of the steps. At this point, the user can select to customize this NIC, add new ones, or remove all.
+
+![Adding NIC](img/3-01.png)
```

Comment:
Primary action button could probably be "Add" rather than "Apply" as well.

Reply:
@andybraren 
Do we toggle between "Add" and "Edit" for both the modal's title and primary button or do we do that only for the title and make the button a generic "Apply"?

Reply:
I’d expect “Add NIC” to have an “Add” button and “Edit NIC” (accessed either in this wizard or after creation) to have a “Save” button rather than “Apply” for both. That would align this Edit modal with existing modals like “Edit Labels” and “Edit Annotations” which also use “Save”.